client/landing/gutenboarding/components/domain-picker/suggestion-item-placeholder.tsx

```diff
@@ -6,9 +6,8 @@ import React, { FunctionComponent } from 'react';
 const DomainPickerSuggestionItemPlaceholder: FunctionComponent = () => {
 	return (
 		<div className="domain-picker__suggestion-item">
```

Comment:
Observation: I would have expected the `placeholder` class to appear here and apply styles to descendants rather than setting that class on descendants as required.

No need to change it here.

Reply:
Yeah that makes more sense too.